cypress/support/commands.ts

```diff
@@ -148,7 +148,7 @@ Cypress.Commands.add(
 );
 
 Cypress.Commands.add('enableShowMultipleInDebugTree' as keyof Chainable, () => {
-  cy.get('[data-cy-open-settings-modal]').click();
-  cy.get('input[data-cy-toggle-show-amount]').click();
-  cy.get('[data-cy-settings-modal-save-changes]').click();
+  cy.get("[data-cy-debug-table='openSettings']").click();
```

Comment:
Deze settings button heeft niks te maken met de table
```suggestion
    cy.get("[data-cy-debug-button='settings']").click();
```